packages/scripts/config/webpack.config.js

```diff
@@ -84,7 +84,10 @@ const config = {
 			new LiveReloadPlugin( {
 				port: process.env.WP_LIVE_RELOAD_PORT || 35729,
 			} ),
-		new DependencyExtractionWebpackPlugin( { injectPolyfill: true } ),
+		// WP_NO_EXTERNALS global variable controls whether scripts' assets get
+		// generated, and the default externals set.
+		process.env.WP_NO_EXTERNALS &&
```

Comment:
I may be missing something, but is the logic inverted here? "No externals" means that the dependency extraction should **not** run, right?

Reply:
Good catch, should I change the name of the variable or apply negation then? :)

Reply:
My suggestion would be to apply negation, but that's my personal preference 🙂

I usually prefer when flags are for non-default behaviour; so if the default behaviour is to use externals, the flag would be for switching to the non-default behaviour, hence "no externals".

If you prefer to avoid negation wording in flag names, you'd need to come up with a different name, like `WP_BUNDLE_EVERYTHING`, but better 😄

Reply:
In this case, it's fine to go with double negation as it's an edge case. It will allow bundling the same code for WP and for the non-WP context with the flag passed.